packages/expo-audio/src/index.ts

```diff
@@ -1 +1,7 @@
+export { PermissionResponse, PermissionExpiration, PermissionStatus } from 'expo-modules-core';
+
 export * from './ExpoAudio';
+
+export * from './Audio.types';
+export * from './AudioModule.types';
+export * from './RecordingConstants';
```

Comment:
Can we list things that we want to export? I'd like to avoid the situation when we add some helper types to and mistakenly export them as public APIs. Some types should stay private.
Also, I think it should be very clear by just reading `index.ts` to see what exactly the package exports.